specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/nova.json

```diff
@@ -0,0 +1,1051 @@
+          "type": "string",
+          "description": "The unique resource identifier of the ARM resource."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the ARM resource.",
+          "minLength": 1,
+          "maxLength": 100
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of Azure resource."
+        },
+        "location": {
+          "type": "string",
+          "description": "The location of the resource group to which the resource belongs."
+        },
+        "tags": {
+          "$ref": "#/definitions/Tags"
+        },
+        "properties": {
+          "type": "object",
+          "description": "Properties of a managed Cassandra cluster.",
+          "properties": {
+            "provisioningState": {
+              "readOnly": true,
+              "description": "(Read only.) The status of the Cosmos DB account at the time the operation was called. The status can be one of following. 'Creating' – the Cosmos DB account is being created. When an account is in Creating state, only properties that are specified as input for the Create Cosmos DB account operation are returned. 'Succeeded' – the Cosmos DB account is active for use. 'Updating' – the Cosmos DB account is being updated. 'Deleting' – the Cosmos DB account is being deleted. 'Failed' – the Cosmos DB account failed creation. 'DeletionFailed' – the Cosmos DB account deletion failed.",
+              "type": "string"
+            },
+            "restoreFromBackup": {
+              "type": "string",
+              "description": "Omit or set to null to create a new, empty cluster. If you are creating  new cluster by restoring a backup, then provide the unique resource identifier of the backup."
+            },
+            "delegatedManagementSubnetId": {
+              "type": "string",
+              "description": "(Can be set only at creation time.) Resource ID of a subnet that this cluster's management service should run in."
+            },
+            "cassandraVersion": {
+              "type": "string",
+              "description": "Which version of Cassandra should this cluster converge to running (e.g., 3.11). When updated, the cluster may take some time to migrate to the new version."
+            },
+            "overrideClusterName": {
+              "type": "string",
+              "description": "(Can be set only at creation time.) Optional value to use for clusterName in cassandra.yaml. If omitted, use the cluster's resource name."
+            },
+            "authenticationMethod": {
+              "type": "string",
+              "description": "Which authentication method Cassandra should use to authenticate clients.",
+              "enum": [
+                "None",
+                "Cassandra"
+              ]
+            },
+            "initialCassandraAdminPassword": {
```

Comment:
>initialCassandraAdminPassword [](start = 13, length = 29)

should be marked with x-ms-mutability and x-ms-secret [https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html#oapi013-secrets-must-be-annotated-with-x-ms-secret-and-x-ms-mutability](https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html#oapi013-secrets-must-be-annotated-with-x-ms-secret-and-x-ms-mutability)